keyboards/mechwild/sugarglider/info.json

```
@@ -0,0 +1,101 @@
+{
+    "manufacturer": "MechWild",
+    "keyboard_name": "Sugar Glider",
+    "maintainer": "kylemccreery",
+    "url": "https://mechwild.com/product/sugar-glider/",
+    "bootloader": "stm32-dfu",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
+        "rgblight": true,
+        "dip_switch": true,
+        "steno": true
+    },
+    "usb": {
+        "vid": "0x6D77",
+        "pid": "0x1710",
+        "device_version": "0.2.0"
+    },
+    "diode_direction": "COL2ROW",
+    "rgblight": {
+        "led_count": 10,
+        "pin": "B5",
+        "max_brightness": 255,
+        "hue_steps": 8,
+        "saturation_steps": 8,
+        "brightness_steps": 8,
+        "animations": {
+            "all": true
```

Could you explicitly define which animations are enabled? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

---

I see this feedback often, and I wonder why "all" is an option in the docs